cogstat/cogstat_chart.py

```diff
@@ -725,6 +725,107 @@ def create_variable_pair_chart(data, meas_lev, x, y, result=None, raw_data=False
         graph = plt.gcf()
     return graph
 
+def create_multiple_variable_chart(data, meas_lev, x, type="raw"):
+    """Draw a chart relating more than two variables displaying raw data
+
+    Parameters
+    ----------
+    data : pandas dataframe
+    meas_lev : {'int', 'ord', 'nom', 'unk'}
+        Measurement level of the variables
+    x : str
+        Name of the x variable.
+    type : {'raw', 'collinearity'}
```

Comment:
if there is no shared code between the two modes, then probably it is better to split the function into two separate functions

Reply:
Function split into two separate funcitons.